uninstall.php

```diff
@@ -11,4 +11,22 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() && 100 >= get_sites( array( 'count' => true ) ) ) {
```

Comment:
I feel like we should check the count condition inside the loop. Otherwise, it will run else part for multisite with more than 100 sites.

Reply:
I would argue we don't need that check for the site count here at all. It's another query to run, which we can avoid. I think we should still handle multisites in the right way, and we can just limit the `get_sites()` query like you do below. That means:
* For networks with 100 or fewer sites, it will wipe all sites.
* For networks with more than 100 sites, it will at least wipe 100 sites.

The latter is not great, but better than what we currently have, while accounting for scalability issues. Also, deleting the option for 100 sites when there are more is still better than falling back to only deleting the option for the current site.

Reply:
Also, let's add a comment here explaining this block.
```suggestion
// For a multisite, delete the option for all sites (however limited to 100 sites to avoid memory limit or timeout problems in large scale networks).
if ( is_multisite() ) {
```